packages/pcap-agent/packaging

```diff
@@ -0,0 +1,30 @@
+set -e
+
```

Comment:
maybe add one section here as a comment saying that we are statically linking libpcap to the pcap-agent binary here.